data/coalition/coalition culture conversations.txt

```diff
@@ -233,3 +233,137 @@ mission "Coalition Folklore: Kimek"
+			`	They appear to be boarding the few ships that weren't damaged enough to have their hulls ruptured. After a few short minutes, they seem to have seized control of the vessels, and are having them follow the warships out of the system.`
+
+
+
+mission "Coalition Departing Trouble - C"
+	minor
+	invisible
+	to offer
+		has "Coalition: First Contact: done"
+		not "assisting lunarium"
+		not "joined the heliarchs"
+		random < 2
+	source
+		government "Coalition"
+	destination "Ring of Friendship"
+	on enter
+		conversation
+			`You depart to find something of a screen of Heliarch warships around <origin>, circling it and approaching all departing civilian ships. It's no different with your <ship>, and soon enough you're hailed by a Heliarch Punisher approaching your position.`
+			`	"Salutations, Captain <last>. Performing a routine inspection, we are. Board you momentarily we shall, to verify the safety of your vessel, then leave you to your business we shall."`
+			`	They sign off before you can respond, and the Punisher readies to board the <ship>, as more warships gather around the scene. Dozens of armed Heliarch agents march into your ship silently, diligently going about checking the bunks, cockpit, engine room...`
+			`	Within a few minutes, they appear satisfied with turning your ship upside down, and all march back to the Punisher. They thank you for "cooperating", and the ships clear the way for you to move out.`
+
+
+
+mission "Coalition Incident: Secret Sky"
+	minor
+	invisible
+	to offer
+		has "Coalition: First Contact: done"
+		random < 3
+	source "Secret Sky"
+	on offer
+		conversation
+			`You see an Arach captain crossing the spaceport again and again, to reach the few people in sight here. He converses with each one briefly, showing something in a small monitor, and then moving on to the next shortly after. After a while, he gets to you.`
+			`	"Captain? A moment if free you are?" He asks scrambling his legs to light up the monitor. It displays a picture of a young Kimek. "A passenger of mine. Stopped by here to drop a few Saryd pilgrims, we had, but after seen them off I had, find this Kimek back in my ship, I couldn't!" He exclaims, pointing to an Arach Transport parked just before the edge of where <origin>'s dense fog cuts off your view. "Seen him anywhere, have you?"`
+			choice
+				`	"Sorry, I haven't. Did you try getting in touch with the local Heliarchs?"`
+				`	"Sorry, but I haven't seen him either."`
+					goto end
+			`	He nods. "Contacted them a few hours ago, I had. Look for my passenger in the restricted sections, I wished to, but allowed there I am not, so figured that ask them for aid, I could." He sighs. "Filed a report and sent a few agents off to look for him, they did, but so far received I have not any word from them."`
```

Comment:
```suggestion
			`	He nods. "Contacted them a few hours ago, I had. Look for my passenger in the restricted sections, I wished to, but allowed there I am not, so figured that ask them for aid, I could." He sighs. "Filed a report and sent a few agents off to look for him, they did, but so far received I have not any word from them."`
```
Is the end of the last sentence properly coalition-ed? I might just be confusing myself tbh, not sure.

Reply:
Changed it to what I think is more common Coal speech.